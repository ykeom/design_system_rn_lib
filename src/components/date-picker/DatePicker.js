import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class DatePicker extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>date picker</Text>
      </View>
    );
  }
}

export default DatePicker;
