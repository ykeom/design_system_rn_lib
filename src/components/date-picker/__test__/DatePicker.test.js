import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DatePicker from '../DatePicker';

describe('DatePicker', () => {
  it('should render', () => {
    const component = shallow(<DatePicker />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
