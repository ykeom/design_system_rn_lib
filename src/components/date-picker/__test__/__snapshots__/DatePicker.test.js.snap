// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    date picker
  </Text>
</View>
`;
